resources/views/tools/database/edit-add-bread.blade.php

```diff
@@ -1,3 +1,12 @@
+<?php
+    $types = [
+        'number' => ['smallint', 'mediumint', 'integer', 'float', 'double', 'decimal'],
+        'date' => ['datetime', 'date'],
+        'textarea' => ['bigtext', 'text', 'mediumtext', 'longtext'],
```

Comment:
Agreed.  Also, the field types listed are DBMS-specific.  Mysql and Postgres for example have different field names.  If we later choose to implement Mongo, MS SQLServer, etc, those will be different as well.